config/config.go

```diff
@@ -70,10 +70,10 @@ type Config struct {
 	AzureClientID     string `name:"azure_client_id" env:"ARM_CLIENT_ID" auth:"azure"`
 	AzureTenantID     string `name:"azure_tenant_id" env:"ARM_TENANT_ID" auth:"azure"`
 
-	// AzureEnvironment (Public, UsGov, China, Germany) has specific set of API endpoints.
+	// [Deprecated] AzureEnvironment (Public, UsGov, China, Germany) has specific set of API endpoints.
```

Comment:
Beyond saying "deprecated", let's also say that they no longer have any effect.

Reply:
ah wait, we do still need AzureEnvironment for cases when no `host` and only `azure_workspace_resource_id` is supplied, we have to call resource management to fetch the workspace url.